gitops-server.dockerfile

```diff
@@ -28,7 +28,6 @@ COPY . /app
 # These are ARGS are defined here to minimise cache misses
 # (cf. https://docs.docker.com/engine/reference/builder/#impact-on-build-caching)
 # Pass these flags so we don't have to copy .git/ for those commands to work
-ARG LDFLAGS="-X localbuild=true"
```

Comment:
Why is this being removed? It's needed in order to correctly inject the `$LDFLAGS` as set in the makefile without copying the entire `.git` directory into the docker file.